app/main/views.py

```diff
@@ -1,13 +1,12 @@
-from flask import render_template
+from flask import render_template, request, flash
 from . import main
 from app import models, db
-
+from forms import NewIncidentForm
 @main.route('/')
```

Comment:
Did flake8 not give an error here? You should put two spaces before the function.